src/components/Sidebar/ChannelsSidebar/ChannelsFooter/index.tsx

```diff
@@ -0,0 +1,25 @@
+import * as Styles from '@components/Sidebar/ChannelsSidebar/ChannelsFooter/styles';
+import React, { useEffect } from 'react';
+import { useStoreActions, useStoreState } from '@state';
+import { ChannelsFooterUserInfo } from '@components/Sidebar/ChannelsSidebar/ChannelsFooter/ChannelsFooterUserInfo';
+import { ChannelsFooterLoginButton } from '@components/Sidebar/ChannelsSidebar/ChannelsFooter/ChannelsFooterLoginButton';
+
+export const ChannelsFooter = () => {
+  const user = useStoreState(state => state.user.data);
+
+  const setShowGuestFormModal = useStoreActions(state => state.ui.setShowGuestFormModal);
+
+  useEffect(() => {
+    if (user) {
+      setShowGuestFormModal(false);
+    }
+  }, [setShowGuestFormModal, user]);
+
+  return (
+    <Styles.ChannelsFooterWrapper>
+      {!user ? <ChannelsFooterLoginButton /> : <ChannelsFooterUserInfo />}
```

Comment:
Imo this is more readable rather than inversing just to change the ternary order

```suggestion
      {user ? <ChannelsFooterUserInfo /> : <ChannelsFooterLoginButton />}
```